plugin/route53/route53.go

```diff
@@ -178,13 +184,15 @@ func maybeUnescape(s string) (string, error) {
 
 		li, ri := i+len(escapeSeq), i+len(escapeSeq)+3
 		if ri > len(s) {
-			return "", fmt.Errorf("invalid escape sequence: '%s%s'", escapeSeq, s[li:])
+			return "", fmt.Errorf("%w: %s%s", errInvalidEscape, escapeSeq, s[li:])
+			//return "", fmt.Errorf("invalid escape sequence: '%s%s'", escapeSeq, s[li:])
 		}
 		// Parse `\\xxx` in base 8 (2nd arg) and attempt to fit into
 		// 8-bit result (3rd arg).
 		n, err := strconv.ParseInt(s[li:ri], 8, 8)
 		if err != nil {
-			return "", fmt.Errorf("invalid escape sequence: '%s%s'", escapeSeq, s[li:ri])
+			return "", fmt.Errorf("%w: %s%s", errInvalidEscape, escapeSeq, s[li:ri])
```

Comment:
%w means we will require a very new Go, don't think its wise to use this just yet